### PR TITLE
Fix path for files in /usr/share

### PR DESCRIPTION
**Pull request by inknos**
Some files got added twice in https://github.com/rpm-software-management/dnf5/commit/f8e861b56a3e7c1b2b81a45f8dec86c94b404c1a

### CODE REVIEW

**Comment by jan-kolarik** [comment]
Maybe I am missing something, but I don't see those files anywhere else in the `.spec` file?

**Review by m-blaha — CHANGES_REQUESTED** [review]

File: dnf5.spec [hunk 1]
```diff
@@ -173,9 +173,6 @@ It supports RPM packages, modulemd modules, and comps groups & environments.
 %{_prefix}/share/dnf5
 %dir %{_sysconfdir}/dnf/dnf5-aliases.d
 %doc %{_sysconfdir}/dnf/dnf5-aliases.d/README
-%dir %{_prefix}/lib/dnf5
-%dir %{_prefix}/lib/dnf5/aliases.d
-%config %{_prefix}/lib/dnf5/aliases.d/compatibility.conf
```
[the remark this answers is not in the document]
Comment (inknos) [reply dnf5.spec:178]: oh, I had the description wrong. The files are indeed duplicate in `/usr/share/dnf5`, which we ship.
originally aliases.d and such were in `/usr/lib` but got moved in `/usr/share`.
The pr intends to remove the duplicate files in `/usr/share/dnf5` but mistakenly I confused the path in the description.

I will update the pr accordingly to what proposed

[the remark this answers is not in the document]
Comment (inknos) [reply dnf5.spec:178]: @m-blaha check it now

[the remark this answers is not in the document]
Comment (inknos) [reply dnf5.spec:190]: @m-blaha 👍🏻

[the remark this answers is not in the document]
Comment (m-blaha) [reply dnf5.spec:190]: Oh, I see, thanks. The clang builds are failing on rawhide again, but that's a different issue.

**Review by m-blaha — APPROVED** [review]

**Pull request merged**